epub33/core/index.html

```diff
@@ -7623,9 +7623,12 @@ <h4>Content document dimensions</h4>
 								</ul>
 								<p>The <code>device-width</code> and <code>device-height</code> values refer to the 100%
 									of the width and height, respectively, of the reading system's [=viewport=].</p>
-								<p>EPUB creators MUST NOT specify duplicate <code>height</code> or <code>width</code>
-									definitions either within a single <code>viewport meta</code> tag or by specifying
-									multiple <code>viewport meta</code> tags.</p>
+								<p>The height and width of the fixed-layout document MUST be defined in the first
```

Comment:
minor editorial suggestion:

s/height and width of the fixed-layout document/height and width definitions/

(it's the height and width of the ICB; also, it's more consistent with when say "REQUIRED height and width definitions")

Reply:
Fixed in the latest commit.